receiver/k8sclusterreceiver/internal/clusterresourcequota/clusterresourcequotas.go

```diff
@@ -0,0 +1,62 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
I didn't actually delete `clusteresourcequoats.go`, and this isn't a whole new file, it's just a rename to fix the misspelling.